resources/views/registerProject.blade.php

```diff
@@ -0,0 +1,218 @@
+					<p>All fields are required for registering a new project.</p>
+				</div>
+				<!-- Project Image -->
+				<div class="projectImg">
+					<img src="https://lorempixel.com/400/400" alt="Project Image" class="ui centered medium circular image"/>
+					<a href="#" class="imgUploader">
+						<button id="uploadImage" class="fluid ui button">Upload Image</button>
+					</a>
+					<input type="file" id="imgInput" style="display:none" multiple accept='image/*'>
+				</div>
+				<!-- Project Name -->
+				<div id ="projectName" class="field">
+					<label for="proyectName">Project Name</label>
+					<input type="text" name="projectName" placeholder="Enter your project's name...">
+				</div>
+				<!-- Team Name -->
+				<div id="teamName" class="field">
+					<label for="teamName">Team Name</label>
+					<input type="text" name="teamName" placeholder="Enter your team's name...">
+				</div>
+				<!-- Category -->
+					<!-- <div id="category" class="field">
+						<label>Category</label>
+						<div class="ui fluid search selection dropdown category">
+							<i class="dropdown icon"></i>
+							<input id ="category" type="hidden" name="country">
+							<div class="default text">Select a category of your project</div>
+							<div class="menu">
+								<div class="item" value="1">Category 1</div>
+								<div class="item" value="2">Category 2</div>
+								<div class="item" value="3">Category 3</div>
+								<div class="item" value="4">Category 4</div>
+							</div>
+						</div>
+					</div> -->
+				<div id ="category" class="field">
+					<label for="category">Category</label>
+					<select name="category" class="ui search dropdown" placeholder="Select your project's category">
+						<option style="color:grey;" value="0">Select your project's category</option>
+						<option value="1">Category 1</option>
+						<option value="2">Category 2</option>
+						<option value="3">Category 3</option>
+					</select>
+				</div>
+				<!-- Skillset -->
+				<div id="skillsets" class="field">
+					<label for="skillsets">Required Skillsets</label>
+					<div class="ui search">
+						<div class="ui icon input">
+							<input name="skillsets" class="prompt" type="text" placeholder="Search and add skillsets...">
+							<i class="search icon"></i>
+						</div>
+						<div class="results">
+						</div>
+					</div>
+				</div>
+				<!-- Milestones -->
+				<div id="milestones" class="field">
+					<label for="milestones">Public Milestones</label>
+					<input type="text" name="milestones" placeholder="e.g. Shipping">
+				</div>
+				<!-- Short Description -->
+				<div id="shortDescription" class="field">
+					<label>Brief Description</label>
+					<textarea rows="2"></textarea>
+				</div>
+				<!-- Long Description -->
+				<div id="longDescription" class="field">
+					<label>Detailed Description</label>
+					<textarea></textarea>
+				</div>
+				<!-- Video Pitch -->
+				<div id="videoPitch" class="field">
+					<label>Video pitch</label>
```

Comment:
Labels should also be consistent: either capitalize the first letter of every word or just the first letter of the first works but keep it consistent.